doc/modules/ROOT/pages/index.adoc

```diff
@@ -6,105 +6,280 @@
+]
+----
+
+At this point you have a Hazelcast cluster exposed with a single LoadBalancer service called `hz-hazelcast`. You can find it's address with the following command.
+
+[source, bash]
 ----
-$ kubectl get pods
-NAME          READY   STATUS    RESTARTS   AGE
-hazelcast-0   1/1     Running   0          2m
-hazelcast-1   1/1     Running   0          2m
-hazelcast-2   1/1     Running   0          2m
+kubectl get service hz-hazelcast
+NAME           TYPE           CLUSTER-IP       EXTERNAL-IP       PORT(S)          AGE
+hz-hazelcast   LoadBalancer   10.108.141.178   10.96.184.178     5701:31434/TCP   5m44s
 ----
 
 [NOTE]
 ====
-If you are using Minikube, you need to execute minikube tunnel now in order to get LoadBalancer External IPs assigned.
+If you are using Minikube, you need to execute `minikube tunnel` now in order to get LoadBalancer External IPs assigned.
 ====
 
-As the first step, you need to start Hazelcast cluster in such a way that each member is exposed with a separate public IP/port. In Kubernetes PODs can be accessed from outside only via services, so the configuration requires creating a separate service (LoadBalancer or NodePort) for each Hazelcast member POD. This can be done manually or using https://metacontroller.app/[Metacontroller] plugin with https://github.com/metacontroller/metacontroller/tree/master/examples/service-per-pod[Service-Per-Pod] Decorator Controller.
+The field `EXERNAL-IP` is the address of your Hazelcast cluster.
+
+=== Connecting with Hazelcast Client
+
+Configure the Hazelcast client with the external address and disable smart routing to use the unisocket connection.
 
 [tabs]
 ====
-Manual::
+
+Java::
 +
 --
+[source, java]
+----
+ClientConfig config = new ClientConfig();
+config.getNetworkConfig().addAddress("<EXTERNAL-IP>")
+                         .setSmartRouting(false);
+----
+--
 
-To create a `loadbalancer` for each running Hazelcast pod you need to run the following command:
+NodeJS::
++
+--
+[source, javascript]
+----
+const { Client } = require('hazelcast-client');
 
-[source, shell script]
+const clientConfig = {
+    network: {
+        clusterMembers: [
+            '<EXTERNAL-IP>'
+        ],
+        smartRouting: false
+    }
+};
+const client = await Client.newHazelcastClient(clientConfig);
 ----
-for pod in $(kubectl get pods -o jsonpath="{.items[*].metadata.name}"); do \
-  kubectl create service loadbalancer ${pod} --tcp=5701 -o yaml --dry-run=client | kubectl set selector --local -f - "statefulset.kubernetes.io/pod-name=${pod}" -o yaml | kubectl create -f -; \
-done
+--
+
+Go::
++
+--
+[source, go]
+----
+import (
+	"log"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+func main() {
+    config := hazelcast.NewConfig()
+	config.ClusterConfig.SetAddress("<EXTERNAL-IP>:5701")
+    config.ClusterConfig.SmartRouting = false
+	client, err := hazelcast.StartNewClientWithConfig(config)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Somehow these lines have different indentation.

Just an idea, but I like to store `config.ClusterConfig` in a variable if I need to set a few things on it, like:
```go
cc := &config.ClusterConfig
cc.SetAddress("<EXTERNAL-IP>:5701")
cc.SmartRouting = false
```